File: java/src/test/java/com/alxarafe/domain/topology/valueobject/LocationRoleTest.java
package com.alxarafe.domain.topology.valueobject;

import com.alxarafe.app.domain.topology.valueobject.LocationRole;
import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.*;

class LocationRoleTest {

    @Test
    void values() {
        assertEquals(LocationRole.PICKING, LocationRole.valueOf("PICKING"));
        assertEquals(LocationRole.RESERVE, LocationRole.valueOf("RESERVE"));
        assertEquals(2, LocationRole.values().length);
    }
}

File: php/tests/Unit/Domain/Topology/ValueObject/LocationRoleTest.php
<?php

declare(strict_types=1);

namespace Tests\Unit\Domain\Topology\ValueObject;

use Alxarafe\App\Domain\Topology\ValueObject\LocationRole;
use PHPUnit\Framework\TestCase;

final class LocationRoleTest extends TestCase
{
    public function testValues(): void
    {
        self::assertSame('PICKING', LocationRole::PICKING->value);
        self::assertSame('RESERVE', LocationRole::RESERVE->value);
    }
}
